packages/lage/RELEASE.md

```diff
@@ -0,0 +1,17 @@
+# Release v2.0.0
+
+Version 2 of lage is finally arriving at the dist-tag of @latest! There had been many, many changes under the hood to make this release truly unique. Here are some of the highlights:
+
+New Features:
+
+* `worker_threads` based worker pool to manage the parallel targets
+* new `worker` target runner that can keep state from task to task to speed up TypeScript, ESLint, etc.
+* worker pool that allows allocation of certain number of workers per task type
+* customizable CLI argument to be passed along to the `npm script` runner
+* weighted targets to allow better utilization of resources for tasks that spawn multiple processes like Jest
+* brand new UI for an local interactive view of work done by workers
+* lage is a bundled and minimized, making the package itself easy to install
+* featured in v1, but expanded in v2: the configuration syntax of pipelines is expanded to allow for more configuration options
+* switched from `yargs` to `commander` has made the CLI arg parsing to be more resilient and help messages more ... helpful 
```

Comment:
If there is better help messages, mention that. Otherwise this probably won't matter to end users.